portfolio/src/main/webapp/index.html

```diff
@@ -79,50 +79,66 @@ <h3>UCSD Triton</h3>
             <div class="row">
                 <div class="column">
                     <div class="content">
-                        <img src="/images/step.jpg" alt="Google STEP Logo" style="width:100%">
+                        <img src="/images/step.jpg" alt="Google STEP Logo" border=1 style="width:100%">
                         <h3>Google STEP</h3>
+                        <h5>June 2020 - Present</h5>
                         <p>
                             Hey, that's right now!
                         </p>
                     </div>
                 </div>
                 <div class="column">
                     <div class="content">
-                        <img src="/images/tse.png" alt="Triton Software Engineering" style="width:100%">
+                        <img src="/images/tse.png" alt="Triton Software Engineering" border=1 style="width:100%">
```

Comment:
Ditto

Reply:
The CSS `content > img` command will take care of borders for all of these images